src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -28,6 +29,54 @@
 
 using namespace ov;
 
+bool inline static need_slice_clamping(std::shared_ptr<ov::op::v0::Parameter> param, ov::element::Type to) {
```

Comment:
Slice is just a one of the sensitive layers 
```suggestion
bool inline static need_clamping(std::shared_ptr<ov::op::v0::Parameter> param, ov::element::Type to) {
```